content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/create-slack-alerts/index.md

```diff
@@ -0,0 +1,223 @@
+```
+
+```
+Initializing provider plugins...
+- Finding latest version of ko-build/ko...
+- Finding latest version of hashicorp/google...
+- Installing ko-build/ko v0.0.7...
+- Installed ko-build/ko v0.0.7 (self-signed, key ID 36A8C1D1056CC508)
+- Installing hashicorp/google v4.61.0...
+- Installed hashicorp/google v4.61.0 (signed by HashiCorp)
+. . .
+```
+
+Now you can generate an execution plan. You will need a few pieces of information to input as variables:
+
+1. `group`: your Chainguard group ID. Use `chainctl iam groups describe <your group> -o json |jq -r '.id'` to retrieve it.
+
+2. `project_id`: your Google Cloud project ID. Use the human readable name for the project as opposed to the numeric identifier.
+
+Once you have all the information gathered, run the `terraform plan` command:
+
+```shell
+terraform plan \
+  -var "group=<chainguard group ID>" \
+  -var "project_id=<gcloud project ID>" \
+  -var "name=enforce-events" \
+  -out enforce-events.plan
+```
+
+You will receive output showing Terraform's execution plan like the following:
+
+```
+. . .
+  # google_cloud_run_service.slack-notifier will be created
+  + resource "google_cloud_run_service" "slack-notifier" {
+      + autogenerate_revision_name = false
+      + id                         = (known after apply)
+      + location                   = "<google location>"
+      + name                       = "enforce-events-slack-notifier"
+      + project                    = "<gcloud project ID>"
+      + status                     = (known after apply)
+. . .
+To perform exactly these actions, run the following command to apply:
+    terraform apply "enforce-events.plan"
+```
+
+Inspect the plan to ensure that the resources that Terraform will create are in line with your Google Cloud project's settings and IAM permissions. Once you are satisfied with the plan, apply it:
+
+```shell
+terraform apply enforce-events.plan
+```
+
+Terraform will run the module and you will receive output like the following:
+
+```
+google_service_account.slack-notifier: Creating...
+google_secret_manager_secret.slack-url: Creating...
+. . .
+Outputs:
+
+secret-command = "echo -n YOUR SLACK WEBHOOK | gcloud --project jamon-chainguard secrets versions add enforce-events-slack-url --data-file=-"
+url = "https://enforce-events-slack-notifier-ntfj7ezpja-uc.a.run.app"
+```
+
+Note down the `url`, and the secret-command.
+
+## Configuring Google Cloud with your Slack Webhook URL
+
+With the output from the `terraform apply` step, you can now run the `echo` command to configure Google Cloud with your Slack webhook URL.
+
+Substitute your webhok URL in place of the `YOUR SLACK WEBHOOK` string and run the following command:
+
+```shell
+echo -n <YOUR SLACK WEBHOOK> | gcloud --project <google project> secrets versions add enforce-events-slack-url --data-file=-
+```
+
+This configures your Google Cloud project with your URL, so that it is available to the demo application when it is invoked on Cloud Run.
```

Comment:
```suggestion
This configures your Google Cloud project with your URL so that it is available to the demo application when it is invoked on Cloud Run.
```